framework/packages/abstract-interface/src/account/manager.rs

```diff
@@ -65,22 +65,31 @@ impl<Chain: CwEnv> Manager<Chain> {
     pub fn replace_api(
         &self,
         module_id: &str,
+        // this field is required if you reinstall same version
+        module_salt: Option<Binary>,
```

Comment:
replace_api is actually replace_adapter and adapters never need salts (as they are already instantiated)